lib/hive/controller/android.rb

```diff
@@ -6,13 +6,16 @@ module Hive
   class Controller
     class Android < Controller
       attr_accessor :devices
+      attr_accessor :attached_devices
 
-      def detect
-        all_attached_devices = get_connected_devices # get all connected devices
-        to_poll = select_devices(all_attached_devices) # select devices to poll
+      def detect(device_type = 'Mobile')
+        # device_type should be either 'Mobile' or 'Tv'
+        @device_type = device_type
```

Comment:
We will need the device_type attribute when we subclass this Android Controller with the AndroidTV controller down the line - I have written this code but it's not live yet. The AndroidTV controller depends on the Android controller.